packages/dev-server-storybook/tsconfig.json

```diff
@@ -1,5 +1,3 @@
-// Don't edit this file directly. It is generated by generate-ts-configs script
-
 {
   "extends": "../../tsconfig.node-base.json",
   "compilerOptions": {
```

Comment:
the lines are gone now :(

Reply:
Yeah let me revert back to a good state.